app/templates/cases/case/subcases/subcase.hbs

```diff
@@ -9,6 +9,7 @@
   <Subcases::SubcaseHeader
     @subcase={{this.model}}
     @onMoveSubcase={{this.refreshSubcases}}
+    @onProposingSubcase={{this.refreshSubcases}}
```

Comment:
In line with naming like `didDelete...`, can we rename to `@onProposedForAgenda` to indicate that the proposal has been done already?

Reply:
true